apps/site/components/__design__/mermaid.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+import { VFile } from 'vfile';
+
+import { compile } from '@/next.mdx.compiler.mjs';
+
+type Props = { children: string };
+
+type Story = StoryObj<Props>;
+type Meta = MetaObj<Props>;
+
+export const Default: Story = {
+  args: {
+    children: `\`\`\`mermaid
```

Comment:
Can you use dedent` ?

Reply:
Dedent is a package that allows you to keep the code indented and then it dedents the children content within the string